docs/dev/development/getting-started.rst

```diff
@@ -321,7 +321,25 @@ the string ``password``. You can log in as any account at
 http://localhost:80/account/login/.
 
 To log in as an admin user, log in as ``ewdurbin`` with the password
-``password``. Due to session invalidation, you may have to login twice.
+``password``. The users below all have TOTP 2FA enabled with a known
+secret. You can generate a TOTP value for logging in using:
+
+.. code-block:: console
+
+    $ make totp
```

Comment:
Excellent. Should this move lower, and either add or replace other totp-cli commands I added yesterday?

Reply:
I don't have too much of an opinion here. I do think the totp-cli bits could be helpful for people who want to add TOTP to other random accounts.

Reply:
moved things around a bit in https://github.com/pypi/warehouse/pull/16263/commits/be64a47311ffa3b9274e4fe1196e75ed26cc3b5f